templates/pages/blog.html

```diff
@@ -15,10 +15,11 @@
 {{/partial}}
 
 {{#partial "page"}}
-    {{> components/common/breadcrumbs breadcrumbs=breadcrumbs}}
-    <section>
+    <section class="max-w-4xl px-2 md:px-4 mx-auto my-8">
```

Comment:
@aleksandraworhacz instead of `max-w-4xl` should it be `max-w-desktop`?

Reply:
in cornerstone theme, the blog page has more narrow content than other pages. That's why I used `max-w-4xl` instead of `max-w-desktop` 

https://cornerstone-light-demo.mybigcommerce.com/journal/

Reply:
@aleksandraworhacz true, we could move the text content to the right in order to use that spacing? I ask because this would be the only place where we'd use a different max width.